engine/classes/Elgg/EmailService.php

```diff
@@ -102,6 +102,7 @@ public function transport(Email $email) {
 		// create the e-mail message
 		$message = new MailMessage();
 		$message->setEncoding('UTF-8');
+		$message->setSender($email->getFrom());
```

Comment:
what does this fix?

Reply:
You can get the sender without having to pull it from `To` array

Reply:
I think you mean the 'from' array? you could also use `getFrom` to get the same data as with `getSender`... or are you trying to retrieve it in a different way?